src/current/molt/molt-fetch.md

```diff
@@ -945,6 +1009,66 @@ molt fetch \
 --non-interactive
 ~~~
 
+### Fail back securely from CockroachDB
+
+The following `molt fetch` command uses `failback` mode to securely replicate changes from CockroachDB back to a MySQL database. This assumes that you migrated data from MySQL to CockroachDB, and want to keep the data consistent on MySQL in case you need to roll back the migration.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+molt fetch \
+--source 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
+--target 'mysql://root:password@localhost/molt?sslcert=.%2fsource_certs%2fclient.root.crt&sslkey=.%2fsource_certs%2fclient.root.key&sslmode=verify-full&sslrootcert=.%2fsource_certs%2fca.crt' \
+--table-filter 'employees, payments' \
+--non-interactive \
+--logging debug \
+--replicator-flags "--tlsCertificate ./certs/server.crt --tlsPrivateKey ./certs/server.key" \
+--mode failback \
+--changefeeds-path 'changefeed-settings.json'
+~~~
+
+- `--source` specifies the connection string of the CockroachDB database to which you migrated.
+- `--target` specifies the connection string of the MySQL database acting as the failback target.
+- `--table-filter` specifies that the `employees` and `payments` tables should be watched for change events.
+- `--replicator-flags` specifies the paths to the server certificate (`--tlsCertificate`) and key (`--tlsPrivateKey`) that correspond to the client certs described in the following item.
+
+	{{site.data.alerts.callout_info}}
+	The default [local, insecure changefeed configuration](#fail-back-to-source-database) requires that you issue `molt fetch` with [`--replicator-flags '--tlsSelfSigned --disableAuthentication'`](#global-flags) instead.
+	{{site.data.alerts.end}}
+
+- `--changefeeds-path` specifies the path to `changefeed-settings.json`, which contains the following setting override:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ json
+	{
+	    "sink_query_parameters": "client_cert={base64 cert}&client_key={base64 key}&ca_cert={base64 CA cert}"
+	}
+	~~~
+
+	`client_cert`, `client_key`, and `ca_cert` are [webhook sink parameters]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-parameters) that must be base64- and URL-encoded (for example, `base64 -i ./client.crt | jq -R -r '@uri'`).
+
+	{{site.data.alerts.callout_success}}
+	For details on the default changefeed settings and how to override them, see [Fail back to source database](#fail-back-to-source-database).
+	{{site.data.alerts.end}}
+
+The preceding `molt fetch` command issues the equivalent [`CREATE CHANGEFEED`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}) command, using the default and explicitly overriden changefeed settings:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE CHANGEFEED FOR TABLE employees, payments 
+  INTO 'webhook-https://host.docker.internal:30004/defaultdb/public?client_cert={base64 cert}&client_key={base64 key}&ca_cert={base64 CA cert}' 
```

Comment:
"host.docker.internal" is for our local testing. For a customer running locally without docker, I think `host.docker.internal` should be `localhost`. This should make it simpler.